apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -16,6 +16,20 @@
  * under the License.
  */
 
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
```

Comment:
Let's update the imports to oxygen-ui

Reply:
Currently, Oxygen UI does not have a Table component. I have created an [issue](https://github.com/wso2/oxygen-ui/issues/226) to track this.